Projektskabelon-master/src/main/java/FunctionLayer/LogicFacade.java

```diff
@@ -45,9 +44,14 @@ public static int getThicknessByID(int id) throws LoginSampleException {
         int thickness = MaterialMapper.getThicknessByID(id);
         return thickness;
     }
+    public static double getPrice(int id) throws LoginSampleException {
+        double price = MaterialMapper.getPrices(id);
+        return price;
+    }
+
 
     public static ArrayList<Integer> getLengthForMaterials(String nameMaterial) throws LoginSampleException {
-        ArrayList<Integer> lengthForMaterials = MaterialMapper.getLengthForMaterials(nameMaterial);
+        ArrayList lengthForMaterials = MaterialMapper.getLengthForMaterials(nameMaterial);
```

Comment:
Der skal forblive et ArrayList <Integer>

Reply:
Der er 2 metoder i MaterialMapper:  public static ArrayList<Integer> getLengths(String name) og
   public static ArrayList getLengthForMaterials(String materialName)
og her har vi blanding af de to XD